ci/cron/check-releases.sh

```diff
@@ -213,6 +227,7 @@ main() (
         echo "[$(date --date=@$SECONDS -u +%H:%M:%S)] $tag: matches record, skipping."
       else
         echo "[$(date --date=@$SECONDS -u +%H:%M:%S)] $tag: does not match records, see above for differences."
+        exit 1
```

Comment:
This is unrelated to this change, but I noticed it missing. There isn't much point in having this check if it doesn't fail on failure.

I did take a look through the logs of the last five runs, and we don't seem to have hit this condition recently (which likely means never).